infoschema/perfschema/tables_test.go

```diff
@@ -292,6 +292,53 @@ func (s *testTableSuite) TestStmtSummaryHistoryTable(c *C) {
 	).Check(testkit.Rows())
 }
 
+// Test events_statements_summary_by_digest_history.
+func (s *testTableSuite) TestStmtSummaryInternalQuery(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int, b varchar(10), key k(a))")
+
+	// We use the sql binding evolve to check the internal query summary.
+	tk.MustExec("set @@tidb_use_plan_baselines = 1")
+	tk.MustExec("set @@tidb_evolve_plan_baselines = 1")
+	tk.MustExec("create global binding for select * from t where t.a = 1 using select * from t ignore index(k) where t.a = 1")
+	tk.MustExec("set global tidb_enable_stmt_summary = 1")
+	tk.MustQuery("select @@global.tidb_enable_stmt_summary").Check(testkit.Rows("1"))
+	defer tk.MustExec("set global tidb_enable_stmt_summary = false")
```

Comment:
```suggestion
	defer tk.MustExec("set global tidb_enable_stmt_summary = ''")
```
Default is true.

Reply:
Why change this, previous test is set false.